File: user-web/src/types/invoices/PaymentMethod.ts
export type PaymentMethod = 'COD' | 'VNPAY' | 'MoMo' | 'CREDIT_CARD' | 'BANK_TRANSFER';

export function getPaymentMethodLabel(method: PaymentMethod): string {
	switch (method) {
		case 'COD':
			return 'Thanh toán khi nhận hàng';
		case 'VNPAY':
			return 'VNPAY';
		case 'MoMo':
			return 'Ví MoMo';
		case 'CREDIT_CARD':
			return 'Thẻ tín dụng';
		case 'BANK_TRANSFER':
			return 'Chuyển khoản ngân hàng';
		default:
			return 'Không xác định';
	}
}

File: api/Models/Payments/PaymentMethod.cs
namespace api.Models.Payments
{
    public enum PaymentMethod
    {
        COD = 1,
        VNPAY = 2,
        MoMo = 3,
        CreditCard = 4,      // Map với 'credit_card'
        BankTransfer = 5     // Map với 'bank_transfer'
    }
}
